scripts/globals/abyssea.lua

```diff
@@ -1098,6 +1098,13 @@ xi.abyssea.onZoneIn = function(player)
     end
 end
 
+xi.abyssea.onEventFinish = function(player, csid, option)
```

Comment:
onEventFinish (even if not used) should have 4 parameters (player, csid, option, npc)

Reply:
same for zone object? 

this is an example for AlTaieu (for example)
![image](https://user-images.githubusercontent.com/1618002/232209733-222d3301-7dca-49de-864a-e254968f6ffc.png)

however, the name may not be correct, could I put onZoneEventFinish?

Reply:
Fair enough, since there is no npc to interact with here

Reply:
is actually the event target, which is some edge cases is not an npc. on occasion it is the player themself. I don't think the zone object is ever it tho. On retail SE usualy uses a stand in dummy NPC for such interactions.